package.json

```diff
@@ -48,6 +47,7 @@
   ],
   "directories": {},
   "dependencies": {
+    "date-fns": "^3.1.0",
```

Comment:
I recommend moving this to peerDependencies without a version ("*") to avoid conflicts with projects that consume this library and use a different version of date-fns.

In addition to the versionless entry in peerDependencies, add a versioned entry in devDependencies to support local development.

Reply:
Thanks, I have updated the package.json to include a date-fns as versionles peer dependency, as well as adding a versioned dev dependency.